File: Mar-18/08-03-18/python/01-Little-monk-and-good-string.py
s = input()

cnt = 0
ans = 0
for i in range(0,len(s)):
    if(s[i]=='a' or
       s[i]=='e' or
       s[i]=='i' or
       s[i]=='o' or
       s[i]=='u' ):
           cnt+=1
    else:
        ans=max(ans,cnt)
        cnt=0
ans=max(ans,cnt)
print(ans)

File: Mar-18/08-03-18/java/01-Little-monk-and-good-string.java
import java.io.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
        String str=input.readLine();
        int len=str.length();
        int ans=0,cnt=0;
        for(int i=0;i<len;i++)
        {
            if(str.charAt(i)=='a' ||
               str.charAt(i)=='e' ||
               str.charAt(i)=='i' ||
               str.charAt(i)=='o' ||
               str.charAt(i)=='u')
               cnt++;
            else
            {
                ans=Math.max(ans,cnt);
                cnt=0;
            }
        }
        ans=Math.max(ans,cnt);
        System.out.println(ans);
    }
}
